src/main_menu.cpp

```diff
@@ -363,7 +355,7 @@ bool game::opening_screen()
     mmenu_refresh_motd();
     mmenu_refresh_credits();
 
-    u = player();
+    g->u = player();
```

Comment:
Here and below: `main_menu` must not create the `player` object for the `game` class. `main_menu` should call public methods of the `game`, which then decides what to do with its badly non-encapsulated variable.